### PR TITLE
API Image create by url

### PR DESCRIPTION
**Pull request by nrw505**

### CODE REVIEW

File: content/v2/images.md [hunk 1]
```diff
@@ -64,6 +67,14 @@ Here's a `curl` example:
       -X POST \
       https://bikeexchange.com.au/api/v2/client/adverts/1/images
 
+or
+
+    curl -i -H 'Authorization: Bearer 471dfba...' \
+      -H 'Content-Type: multipart/form-data' \
+      -F image_url=https://my.site/images/my_image.jpg \
```
Comment (mipearson) [inline content/v2/images.md]: `-F` loads a file from the disk and passes it in the body. This is probably not what you meant.

Comment (nrw505) [reply content/v2/images.md]: no, it doesn't. `-F` sets a field value. `-F field=@/tmp/file.jpg` sends `field` as a file attachment. `-F field=</tmp/file.jpg` sends `field` as a normal input field whose value is the contents of the file, and `-F field=value` sends `field` with a value of `value`

Comment (mipearson) [reply content/v2/images.md]: Don't need the multipart either. Looking at our other docs, we don't have curl-based examples for actual data submits.

Comment (mipearson) [reply content/v2/images.md]: Oops.

Comment (nrw505) [reply content/v2/images.md]: Yep, well remove the multipart/form-data from the example

**Pull request merged**